PlaceRouteHierFlow/router/RawRouter.cpp

```diff
@@ -180,13 +180,10 @@ std::vector<std::set<RouterDB::point, RouterDB::pointXYComp>> RawRouter::Findset
 };
 
 std::vector<std::set<RouterDB::point, RouterDB::pointXYComp>> RawRouter::Plist2Set(std::vector<std::vector<RouterDB::point>>& plist) {
-  std::vector<std::set<RouterDB::point, RouterDB::pointXYComp>> Sets;
+  std::vector<std::set<RouterDB::point, RouterDB::pointXYComp>> Sets(plist.size());
```

Comment:
you could use; `Sets.reserve(plist.size());`
and then 
```
for (const auto& p : plist) {
Sets.emplace_back(std::set<RouterDB::point, RouterDB::pointXYComp>(p.begin(), p.end());
}
```

Reply:
thanks, emplace_back() is more efficient